### DISCUSSION TITLE
PnP-Search Box web part - question instead of an issue > struggling to find the syntax (if possible)

### DISCUSSION THREAD

**Discussion by EricDeferm01** [microsoft-search/pnp-modern-search#3676] [General]
Log requests as ideas in the discussion forum.
I've put the PnP-Search Box web part on a modern SP Page and would like to retrieve the text input from a user to launch the search term againts a managed property (RefinableString80).
When performing a manual search in a Doc library using the search box **RefinableStringi80:Feldspar** at the top, the URL takes the following format **...?view=7&q=RefinableString81%3AFeldspar** after launching the search (which returns the expected results).
When using the PnP-Search Box web part on a page I can't find a proper way to obtain the same result,
I've tried multiple combinations, yet either the entered search term is not coming true or the URL syntax is different > see image.
How to configure the 'Parameter name' o/t PnP-SB web part to obtain the same URL syntax as described above?
Tx.
![PnPSearchBoxQWPURL](https://github.com/microsoft-search/pnp-modern-search/assets/47383152/12f62b10-597d-4d59-aea5-3df53071d11a)

**Comment by kasperbolarsen** [comment]
Hi, I might have misunderstood the issue, but let's see:
If you delete the Search Box web part and add it again ( just to reset back to default), and then add the result web part , and finally update the query text to RefinableString81:{searchTerms}
The final step is to connect the search box to the results web part:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/65a7b1f1-b977-4103-8925-3e30d21ee9fc)


When you enter a value from the term set in the search box, it should work just fine.
I hope it is something like this you were looking for.
